src/components/header-card-action-buttons/header-card-action-buttons.tsx

```diff
@@ -25,7 +44,14 @@ export const HeaderCardActionButtons: FC<Props> = ({ asset }) => {
         onPress={() => navigate(ModalsEnum.Receive, { asset })}
       />
       <Divider size={formatSize(8)} />
-      <ButtonMedium title="SEND" iconName={IconNameEnum.ArrowUp} onPress={() => navigate(ModalsEnum.Send, { asset })} />
+      <View style={styles.buttonContainer} onTouchStart={onTouchStart}>
```

Comment:
replace `onTouchStart` with beautiful short return anonymous function
```
onTouchStart={() => disableSendAsset && showErrorToast(`Can't send ${asset.symbol}`, errorMessage)}
```

Reply:
Sorry, it doesn't work for mysterious reasons, you can check it yourself

Reply:
try to wrap it with void
`onTouchStart={() => void(disableSendAsset && showErrorToast(`Can't send ${asset.symbol}`, errorMessage))}`. 
 If this woult not work, let it be like this